### PR TITLE
issue/2   on remove liquidity amounts does not update after approve

### PR DESCRIPTION
**Pull request by silvertoml**
i fixed decimal point error on remove liquidity amount after approve

### CODE REVIEW

File: apps/dex/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "@xstyled/styled-components": "^3.8.0",
     "big.js": "^6.2.1",
     "bignumber.js": "^9.1.2",
+    "dotenv": "^16.4.5",
```
Comment (fredev) [inline apps/dex/package.json:25]: react does not need dotenv to be installed, please remove the dependency

File: apps/dex/src/pages/RemoveLiquidity.tsx [hunk 2]
```diff
@@ -46,8 +46,11 @@ export function RemoveLiquidity() {
         autoHideDuration: 3000,
         variant: 'success',
       });
-      allowance.refetch();
+      allowance.refetch();
```
Comment (fredev) [inline apps/dex/src/pages/RemoveLiquidity.tsx:49]: this line have extra spaces at the end of the line after the ;

File: apps/dex/src/pages/RemoveLiquidity.tsx [hunk 3]
```diff
@@ -46,8 +46,11 @@ export function RemoveLiquidity() {
         autoHideDuration: 3000,
         variant: 'success',
       });
-      allowance.refetch();
+      allowance.refetch();      
     }
+    console.log("remove.amountA", remove.amountA);
```
Comment (fredev) [inline apps/dex/src/pages/RemoveLiquidity.tsx:51]: remove both console.log

**Review by fredev — CHANGES_REQUESTED** [review]

File: apps/dex/src/pages/RemoveLiquidity.tsx (hunk 3 above)
Comment (silvertoml) [reply apps/dex/src/pages/RemoveLiquidity.tsx:51]: got it

File: apps/dex/src/pages/RemoveLiquidity.tsx (hunk 2 above)
Comment (silvertoml) [reply apps/dex/src/pages/RemoveLiquidity.tsx:49]: got it

File: apps/dex/package.json (hunk 1 above)
Comment (silvertoml) [reply apps/dex/package.json:25]: got it

**Comment by silvertoml** [comment]
problem mentioned above
it is resolved

**Review by fredev — APPROVED** [review]

**Pull request merged**